cfgov/jinja2/v1/_includes/macros/util/format/datetime.html

```diff
@@ -14,9 +14,7 @@
 
    datetime: A datetime object.
 
-   timezone (optional): Whether to include the timezone or not.
-                        Provide '' to default to 'America/New_York',
-                        otherwise provide actual timezone.
+   timezone (optional): Timezone string value. Default is none.
```

Comment:
Minor: I started moving the `(Optional)` bit to the beginning of the text description so the values wouldn't get so far padded out when there were optional parameters. See https://github.com/cfpb/cfgov-refresh/pull/2032